src/System.Net.Http/src/HttpDiagnosticsGuide.md

```diff
@@ -90,8 +90,8 @@ Otherwise, `Activity.Current` represent some 'parent' activity (presumably incom
 7. `DiagnosticListener.Write("System.Net.Http.HttpRequestOut.Stop", new {Response, RequestTaskStatus})` - notifies that activity (outgoing request) is stopping
 
 # Non-Activity events (deprecated)
-There are two events System.Net.Http.Request and System.Net.Http.Response, currently are also emited for compatibility purposes.  
-They are redundant with the System.Net.Http.HttpRequestOut start and stop events (but do not set Activity.Current and follow activity conventions (start/stop)
+There are two events `System.Net.Http.Request` and `System.Net.Http.Response`, currently are also emited for compatibility purposes.  
+They are redundant with the `System.Net.Http.HttpRequestOut.Start` and `System.Net.Http.HttpRequestOut.Stop` events (but do not set Activity.Current and follow activity conventions (start/stop)
```

Comment:
It seems like you have a missing `)` and that `Activity.Current` can be quoted as well.